ghost/members-api/lib/repositories/member.js

```diff
@@ -967,15 +993,30 @@ module.exports = class MemberRepository {
             const context = options?.context || {};
             const source = this._resolveContextSource(context);
 
-            // Notify paid member subscription start
-            if (this._labsService.isSet('emailAlerts') && ['member', 'api'].includes(source)) {
-                await this.staffService.notifyPaidSubscriptionStart({
-                    member: member.toJSON(),
-                    offer: offer ? this._offerRepository.toJSON(offer) : null,
-                    tier: ghostProduct?.toJSON(),
-                    subscription: subscriptionData
-                }, {transacting: options.transacting, forUpdate: true});
-            }
+            const offerData = offer ? {
+                name: offer.name.value,
+                amount: offer.amount.value,
+                type: offer.type.value,
+                durationInMonths: offer.duration.value.months,
+                duration: offer.duration.value
+            } : null;
+
+            DomainEvents.dispatch(PaidSubscriptionStartedEvent.create({
```

Comment:
If we switch to use the `SubscriptionCreatedEvent` you'll notice that we just pass id's through - which I know is a bit of a PITA, but I think it's the best way to go until we've really nailed down this event pattern.

Reply:
My concern with passing IDs only is that it's not just one data object that we are fetching, we'll need to make requests to fetch multiple bits like the associated offer/tier/member other than the subscription, which is a lot of overhead for values that we already have 🤔

Reply:
Gonna switch to using IDs for now so we can come back and clean this up later with a proper data pattern